webcompat/helpers.py

```diff
@@ -444,6 +444,13 @@ def wrapped_func(*args, **kwargs):
                 file_path = FIXTURES_PATH + request.path
             if not file_path.endswith('.json'):
                 file_path = file_path + '.json'
+            if not os.path.exists(file_path):
```

Comment:
If I can solve the client XHR request, we can remove this hack (I'm pretty sure).

Reply:
I decided to live with this hack, it's just mocking test code. The alternative is to move around a bunch of fixture files, and it doesn't seem worth the effort. 🤷